.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy and Publish ReFrame to PyPI
+
+on:
+  release:
+    types: [released]
```

Comment:
Does this cover pre-releases too?

Reply:
No, the various alternatives are given here: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

Reply:
No, the alternatives types are given here: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

Reply:
You should then add `prereleased` here, too.